src/runtime_src/xdp/profile/plugin/ocl/ocl_power_profile.cpp

```diff
@@ -10,6 +10,7 @@ OclPowerProfile::OclPowerProfile(xrt::device* xrt_device,
     target_device = xrt_device;
     target_xocl_plugin = xocl_plugin;
     target_unique_name = unique_name;
+    output_file_name = "ocl_power_profile_" + target_unique_name + ".csv";
```

Comment:
Make this:
output_file_name = "power_profile_" + target_unique_name + ".csv";

Reply:
Agreed. OCL shouldn't be there